extensions/mailer/runtime/src/main/java/io/quarkus/mailer/runtime/MailConfig.java

```diff
@@ -116,6 +116,17 @@ public class MailConfig {
     @ConfigItem(defaultValue = "OPTIONAL")
     public String startTLS;
 
+    /**
+     * Enable DKIM signature verification.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean enableDkim;
```

Comment:
Move this flag to the `DkimSignOptionsConfig`, making the flag to enable it: `quarkus.mailer.dkim.enable=true`